ui/src/components/policy/PolicyRow.js

```diff
@@ -88,27 +198,315 @@ export default class PolicyRow extends React.Component {
         }
     }
 
+    onSetActive() {
+        this.api
+            .setActivePolicyVersion(
+                this.props.domain,
+                this.props.name,
+                this.state.version,
+                this.props._csrf
+            )
+            .then(() => {
+                this.props.reloadPolicyVersionsFunc(
+                    'setActivePolicyVersion',
+                    false,
+                    `${this.props.name}` + '-' + `${this.state.version}`
+                );
+            })
+            .catch((err) => {
+                this.setState({
+                    errorMessage: RequestUtils.xhrErrorCheckHelper(err),
+                });
+            });
+    }
+
+    expandVersions() {
+        let expanded = this.state.expanded;
+        this.setState({
+            rowInVersionGroup: !expanded ? 'first' : 'single',
+            enableDelete: expanded || !this.state.isActive,
+            expanded: !expanded,
+            newPolicy: false,
+        });
+    }
+
+    closeModal() {
+        this.setState({ showSuccess: null });
+    }
+
+    reLoadPolicyVersions(type, showSuccess, successMessage, closeModalsFunc) {
```

Comment:
This will reload the policy versions and will update the state of the active version.